wp-content/themes/missilethreat/inc/template-tags.php

```diff
@@ -409,4 +409,106 @@ function missiledefense_system_terms() {
 			printf( '%1$s', $html);
 		}
 	}
-endif;
\ No newline at end of file
+endif;
+
+if ( ! function_exists( 'missiledefense_defsys_us' ) ) :
+	/**
+	 * Returns links to U.S. Defense Systems and their icons.
+	 *
+	 */
+	function missiledefense_defsys_us() {
+		$args = array(
+			'post_type' => 'systems',
+			'numberposts' => -1,
+			'orderby' => 'post_title',
+			'order' => 'ASC',
+			'exclude' => array(4087, 4112, 4094, 4103, 4108, 7496, 7492, 7494, 7490)
```

Comment:
I would think about if there's a better system to filter these posts. Right now, we would need to update the theme any time they added something that needed to be excluded, which isn't very sustainable.

My recommendation would be to add an ACF that categorizes the defsys into the 3 groups, and then filter based on that meta value.